Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -850,7 +858,7 @@ - (void)showProtectedDialog {
 
 - (void)sectionEditorFinishedEditing:(SectionEditorViewController*)sectionEditorViewController {
     [self dismissViewControllerAnimated:YES completion:NULL];
-    [self fetchArticleIfNeeded];
+    [self fetchArticleForce:YES];
```

Comment:
nit: pick which layer of this "abstraction" should be used: either the high level methods (i.e. `fetch` or `fetchIfNeeded`) or the underlying `fetch:(BOOL)force`. to be extra explicit, you can prefix the "internal" method w/ an underscore: `_fetchArticle:(BOOL)`